packages/w/wren/xmake.lua

```diff
@@ -0,0 +1,79 @@
+package("wren")
+    set_homepage("http://wren.io")
+    set_description("The Wren Programming Language. Wren is a small, fast, class-based concurrent scripting language.")
+    set_license("MIT")
+
+    add_urls("https://github.com/wren-lang/wren/archive/refs/tags/$(version).tar.gz",
+             "https://github.com/wren-lang/wren.git")
+
+    add_versions("0.4.0", "23c0ddeb6c67a4ed9285bded49f7c91714922c2e7bb88f42428386bf1cf7b339")
+
+    on_install(function (package)
+        local configs = {}
+        io.writefile("xmake.lua", format([[
+            local kind = "%s"
+            add_rules("mode.debug", "mode.release")
+            target("wren")
+                set_kind(kind)
+                add_headerfiles("src/include/*.h", "src/vm/*.h", "src/optional/*.h")
+                add_includedirs("src/include", "src/vm", "src/optional")
+                add_files("src/vm/*.c", "src/optional/*.c")
+
+                if is_mode("debug") then
+                    add_defines("DEBUG")
+                elseif is_mode("release") then
+                    add_defines("NDEBUG")
+                end
+
+        ]], package:config("shared") and "shared" or "static"))
+        if package:config("shared") then
+            configs.kind = "shared"
+        end
```

Comment:
This isn't necessary, xmake already passes kind

Reply:
fixed